### PR TITLE
implemented changes

### PR DESCRIPTION
**Pull request by brookwander**
Changes made to the desorption and reconstruction detection algorithms to make them depended on the covalent radii, rather than length thresholds.

### CODE REVIEW

**Review by mshuaibii — APPROVED** [review]

File: ocdata/test/flag_anomaly.py [hunk 1]
```diff
@@ -1,104 +1,120 @@
...
+            cutoff_multiplier (float, optional): cushion for small atom movements when assessing
+                atom connectivity
+
         Returns:
-                    matrix     The connectivity matrix of the atoms object.
+            (np.ndarray): The connectivity matrix of the atoms object.
         """
-        cutoff = natural_cutoffs(atoms)
-        neighborList = neighborlist.NeighborList(cutoff, self_interaction=False, bothways=True)
+        cutoff = list(np.array(natural_cutoffs(atoms)) * cutoff_multiplier)
```
Comment (wood-b) [inline ocdata/test/flag_anomaly.py:114]: Any reasons we didn't set the `mult` variable that is part of natural_cutoffs method for this?

File: ocdata/test/flag_anomaly.py [hunk 2]
```diff
@@ -1,104 +1,120 @@
...
+        neighborList = neighborlist.NeighborList(
```
Comment (wood-b) [inline ocdata/test/flag_anomaly.py:115]: Maybe change this variable name to nbr_list (lots of variations of neighborlist that are part of the ase method), ase_neighbor_list, or neighbor_list.

**Review by wood-b — COMMENTED** [review]
A few minor comments otherwise looks good to me.

File: ocdata/test/flag_anomaly.py (hunk 1 above)
Comment (brookwander) [reply ocdata/test/flag_anomaly.py:114]: didnt know that existed

File: ocdata/test/flag_anomaly.py (hunk 2 above)
Comment (brookwander) [reply ocdata/test/flag_anomaly.py:115]: that makes sense!

File: ocdata/test/flag_anomaly.py [hunk 3]
```diff
@@ -1,104 +1,120 @@
...
 class DetectTrajAnomaly:
-    def __init__(self, init_atoms, final_atoms, atoms_tag):
+    def __init__(
+        self,
+        init_atoms,
+        final_atoms,
+        atoms_tag,
+        final_slab_atoms=None,
+        surface_change_cutoff_multiplier=1.5,
+        desorption_cutoff_multiplier=1.3,
```
Comment (janiceblue) [inline ocdata/test/flag_anomaly.py:14]: Let's change the default to 1.5 as discussed last week

**Pull request merged**